File: code/0653/653.钞票.py
a = float(input())
b = [100, 50, 20, 10, 5, 2, 1]
print(int(a))
cnt = 0

for i in b:
    while a // i != 0:
        cnt += 1
        a -= i
    print("%d nota(s) de R$ %d,00" % (cnt, i))
    cnt = 0

File: code/0653/653.钞票.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n = sc.nextInt();
        final int[] a = {100, 50, 20, 10, 5, 2, 1};
        System.out.println(n);

        for (int i = 0; i < 7; ++i) {
            int cnt = n / a[i];
            System.out.printf("%d nota(s) de R$ %d,00\n", n / a[i], a[i]);
            n %= a[i];
        }
    }
}
